ee/api/test/test_organization.py

```diff
@@ -203,16 +204,23 @@ def test_feature_available_self_hosted_has_license(self):
             self.organization.refresh_from_db()
             self.assertTrue(self.organization.is_feature_available("whatever"))
             self.assertFalse(self.organization.is_feature_available("feature-doesnt-exist"))
+        License.PLANS = current_plans
```

Comment:
this is going to be annoying for whoever needs to work on this later 😅 - as we are doing the revert only when the test is successful, vs. being in a context manager / beforeEach-afterEach block.

Will look into the `Roles` circular imports tomorrow 👀

Reply:
yeaaaa i didn't know how else to get around this, would def like some suggestions here !